slack/ecr-scanner-notifier/variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "project" {
+  type = string
+}
+
+variable "slack_webhook_url" {
+  type = string
+}
```

Comment:
Can we add a validation that fails, if the URL does not start with `https` to enforce SSL?